content/en/docs/developer_guide/flows/manage_flows/_index.md

```diff
@@ -54,6 +54,18 @@ To edit a flow using the {{% variables/apibuilder_prod_name %}} user interface:
 ![close_caution](/Images/close_caution.png)
 1. Select **Save and exit** to save the flow changes and return to the API Lists page. Select **Discard changes** to discard the unsaved changes to the flow and return to the API Lists page.
 
+## Delete flows
+To delete a flow using the {{% variables/apibuilder_prod_name %}} user interface:
+
+1. Navigate to the flow you wish to delete (through **API Doc & Test** or **Flows**) and open the flow for editing.
+1. If the flow can be deleted, you will see a red **trash can** icon next to the flow name.
+![delete_flow](/Images/delete_flow.png)
+1. Select the **trash can** icon and confirm that you want to delete the flow. The Flow delete caution window is displayed.
+![delete_flow_confirm](/Images/delete_flow_confirm.png)
+1. Select **Proceed** to delete the flow. Select **Cancel** to return to the API Orchestration user interface. If **Proceed** is selected, the server is restarted, and the flow is deleted. Once the server restart is completed, you will be redirected to the previous screen.
```

Comment:
**question**: "API Orchestration"?  Are we using that terminology?  Shouldn't this be "flow editor"?

Reply:
Yeah, it was more for consistency. the whole page needs rewriting since it's very specific to endpoints and doesn't even mention the flows tab.

I've rewritten the section now and cleaned it up a bit with feedback from Andrea when i asked about "trash can".